cpp/density/RDF.cc

```diff
@@ -50,54 +50,13 @@ RDF::RDF(float r_max, float dr, float r_min) : util::NdHistogram(), m_r_max(r_ma
     {
         float r = float(i) * m_dr + m_r_min;
         float nextr = float(i + 1) * m_dr + m_r_min;
-        m_r_array.get()[i] = 2.0f / 3.0f * (nextr * nextr * nextr - r * r * r) / (nextr * nextr - r * r);
+        m_r_array.get()[i] = r + m_dr/2;
```

Comment:
I am planning to remove `m_dr` later, and have everything framed in terms of a number of bins.
```suggestion
        m_r_array.get()[i] = (r + nextr)/2;
```